### PR TITLE
Upgrade gradle plugin and intellij version to 2021.2

### PR DESCRIPTION
**Pull request by ice1000**
I made some minimal efforts to support 2021.2

### CODE REVIEW

**Comment by ice1000** [comment]
[Arend-1.6.1.zip](https://github.com/JetBrains/intellij-arend/files/6945095/Arend-1.6.1.zip)

Here's the version I built

File: build.gradle.kts [hunk 1]
```diff
@@ -54,22 +54,22 @@ idea {
...
-    version = "2021.1.2"
-    pluginName = "Arend"
-    updateSinceUntilBuild = true
-    instrumentCode = true
-    setPlugins("yaml", "java", "IdeaVIM:0.67")
+    version.set("2021.2")
+    pluginName.set("Arend")
+    updateSinceUntilBuild.set(true)
+    instrumentCode.set(true)
+    plugins.set(listOf("yaml", "java", "IdeaVIM:0.67"))
```
Comment (valis) [inline build.gradle.kts:61]: This doesn't work for some reason. It cannot find the `set` method.

Comment (valis) [reply build.gradle.kts:61]: Updating gradle helped I think

**Comment by ice1000** [comment]
[Arend-1.6.1.zip](https://github.com/JetBrains/intellij-arend/files/6986436/Arend-1.6.1.zip)

Updated with latest intellij-arend changes

**Pull request closed without merging**